extensions/amp-list/0.1/test/test-amp-list-container.js

```diff
@@ -72,6 +73,14 @@ describes.realWin(
       env.sandbox.stub(list, 'fetchList_').returns(Promise.resolve());
       list.element.applySize = () => {};
       list.buildCallback();
+
+      lockHeightSpy = env.sandbox.spy(list, 'lockHeightAndMutate_');
+      unlockHeightSpy = env.sandbox.spy(list, 'unlockHeightInsideMutate_');
+    });
+
+    afterEach(() => {
+      expect(lockHeightSpy.notCalled).to.be.true;
+      expect(unlockHeightSpy.notCalled).to.be.true;
```

Comment:
Nit: Can do this using sinon-chai for a slightly cleaner line:
```suggestion
      expect(unlockHeightSpy).not.called;
```